pycbc/inference/prior.py

```diff
@@ -28,11 +28,168 @@
 
 import numpy
 import scipy.stats
+from ConfigParser import Error
 
 #
 #   Distributions for priors
 #
-class Uniform(object):
+
+# helper function for getting bounds from config file
+def get_param_bounds_fromcp(cp, section, tag, param):
```

Comment:
Can this be renamed? We have a couple other functions that end in ``_from_config`` already, perhaps something like ``get_params_bounds_from_config`` or ``get_bounds_from_config``?

At the very least it should end in ``_from_cp`` instead of ``_fromcp`` I think.